[Tutorials] Continue moving metadata to frontmatter (#1315)
timescale/docs: timescaledb/tutorials/grafana/installation.md

---
title: Set up TimescaleDB and Grafana

> excerpt: Set up Managed Service for TimescaleDB with Grafana to visualize your data

Reviewer: ```suggestion
excerpt: Use Grafana to visualize your data on Managed Service for TimescaleDB
```

> keywords: [Grafana, visualizations, analytics]

Reviewer: ```suggestion
keywords: [Grafana, visualization, analytics]
```
